app/themes/3column/ui/app/route/routes.js

```diff
@@ -0,0 +1,105 @@
+(function () {
+  'use strict';
+
+  angular.module('app')
+    .run(['loginService', function(loginService) {
+      loginService.protectedRoutes(['root.search', 'root.create', 'root.profile']);
+    }])
+    .config(Config);
+
+  Config.$inject = ['$stateProvider', '$urlMatcherFactoryProvider',
+    '$urlRouterProvider', '$locationProvider'
+  ];
+
+  function Config(
+    $stateProvider,
+    $urlMatcherFactoryProvider,
+    $urlRouterProvider,
+    $locationProvider) {
+
+    $urlRouterProvider.otherwise('/');
+    $locationProvider.html5Mode(true);
+
+    function valToFromString(val) {
+      return val !== null ? val.toString() : val;
+    }
+
+    function regexpMatches(val) { // jshint validthis:true
+      return this.pattern.test(val);
+    }
+
+    $urlMatcherFactoryProvider.type('path', {
+      encode: valToFromString,
+      decode: valToFromString,
+      is: regexpMatches,
+      pattern: /.+/
+    });
+
+    $stateProvider
+      .state('root', {
+        url: '',
+        // abstract: true,
+        templateUrl: 'app/root/root.html',
+        controller: 'RootCtrl',
+        controllerAs: 'ctrl',
+        resolve: {
+          user: function(userService) {
+            return userService.getUser();
+          }
+        }
+      })
+      .state('root.landing', {
+        url: '/',
+        templateUrl: 'app/landing/landing.html'
+      })
+      .state('root.search', {
+        url: '/search',
+        templateUrl: 'app/search/search.html',
+        controller: 'SearchCtrl',
+        controllerAs: 'ctrl'
+      })
+      .state('root.create', {
+        url: '/create',
+        templateUrl: 'app/create/create.html',
+        controller: 'CreateCtrl',
+        controllerAs: 'ctrl',
+        resolve: {
+          stuff: function() {
+            return null;
+          }
+        }
+      })
+      .state('root.view', {
+        url: '/detail{uri:path}',
+        params: {
+          uri: {
+            squash: true,
```

Comment:
Did you take fresh copies from app/templates/ for the theme folders? I am pretty sure we removed the squash: true for instance, and there are probably several other changes..